jac-uk/admin
# #2032 Remove exercise 'change stage' link

## What's included?
Removes the defunct 'Change Stage' link on the exercise overview page, which is redundant functionality.

Closes #2032  

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
Check that the 'Change stage' link no longer appears on the Exercise Overview page

**Before**
https://admin-develop.judicialappointments.digital/exercise/Acfb8QaFKamIAjKFA9S3/details/overview
<img width="917" alt="image" src="https://github.com/jac-uk/admin/assets/8524401/519935a8-28c4-46bf-b9da-82f9dc3891e3">

**After**
https://jac-admin-develop--pr2090-feature-2032-ejkc5bh4.web.app/exercise/Acfb8QaFKamIAjKFA9S3/details/overview
<img width="909" alt="image" src="https://github.com/jac-uk/admin/assets/8524401/3b84f706-05f5-42b6-bbf6-230b01aa480d">


## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
